### PR TITLE
Modify the form label

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [x]
| BC breaks? (use the c.x branch)        | [x]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

The form label and tooltip were modified. 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new campaign or use an existing one
3. Add an email to the campaign 
4. Select the "in relative time period" option
5. New form label here
image

image

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11166"></a>

### CODE REVIEW

**Review by npracht — COMMENTED** [review]
👍🏽 good for me :)

File: app/bundles/CampaignBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -77,11 +77,11 @@ mautic.campaign.form.type="Execute this event..."
 mautic.campaign.form.type_inaction="Execute this event if the contact does not take action..."
 mautic.campaign.form.type.date="at a specific date/time"
 mautic.campaign.form.type.date_inaction="by a specific date/time"
-mautic.campaign.form.type.help="This event will be executed for newly added contacts to the campaign even if the scheduled date is in the past. For scheduled emails, we recommend you unpublish it if you would not like contacts to receive it after that date."
+mautic.campaign.form.type.help="This event will be executed for newly added contacts to the campaign even if the scheduled date is in the past. For scheduled emails, we recommend you unpublish it if you would not like contacts to receive it after that date. All dates are in the contacts timezone."
```
Comment (RCheesley) [inline app/bundles/CampaignBundle/Translations/en_US/messages.ini]:
```suggestion
mautic.campaign.form.type.help="This event will be executed for newly added Contacts to the Campaign, even if the scheduled date is in the past. For scheduled Emails, it is recommended that you unpublish the Email if you would not like Contacts to receive it after that date. All dates are in the Contact's timezone."
```

**Comment by RCheesley** [comment]
Small grammar suggestions to bring it in line with our styleguide.

**Review by RCheesley — CHANGES_REQUESTED** [review]

**Comment by volha-pivavarchyk** [comment]
@RCheesley Thanks! I updated the message.

**Review by escopecz — APPROVED** [review]

**Pull request merged**